packages/create-pantheon-decoupled-kit/src/templates/next-drupal-search-api-addon/app/search-input.jsx

```diff
@@ -0,0 +1,39 @@
+import { useRouter } from 'next/navigation';
+import { useState } from 'react';
+
+const SearchInput = () => {
+	const [searchQuery, setSearchQuery] = useState('');
+
+	const router = useRouter();
+
+	const onSearch = (event) => {
+		event.preventDefault();
+
+		if (typeof searchQuery !== 'string') {
+			return;
+		}
```

Comment:
This can probably just check for `searchQuery`
```suggestion
		if (!searchQuery) {
			return;
		}
```